util/config/gather-cray-prgenv-arguments.bash

```diff
@@ -32,6 +32,14 @@ module list --terse 2>&1 | grep -q PrgEnv
 if [ $? != 0 ]
 then
   module load PrgEnv-gnu
+else
+  module list --terse 2>&1 | grep -q PrgEnv-gnu
+  if [ $? != 0 ]
+  then
+    # Unload another PrgEnv module and then load PrgEnv-gnu
+    module unload `module list --terse 2>&1 | grep PrgEnv`
+    module load PrgEnv-gnu
+  fi
```

Comment:
Maybe change all of this logic to:

```bash
existing_prgenv=$(module list --terse 2>&1 | grep PrgEnv-)
if [ -n "${existing_prgenv}" ]; then
  module unload "${existing_prgenv}"
fi
module load PrgEnv-gnu
```

Reply:
I didn't want to re-load PrgEnv-gnu if it's already loaded (since it will make the compiler slower).

Reply:
But, I can still use something that looks more like your suggestion.